docs/Manual/04 Fusing.md

```diff
@@ -117,70 +116,72 @@ for i in range(0, 16, 4):
 
-However, note that not every fusing operation creates a safe schedule. For example, imagine that we had fused `schedule0` and `schedule1` differently:
+However, not every fusing operation creates a safe schedule. For example, consider a scenario where we fused `schedule0` and `schedule1` differently:
 ```python
 # Reorder schedule1 before fusing
 schedule1.reorder(j1, i1)
 # Fuse schedule0 with the reordered schedule1
 schedule_t = acc.fuse(schedule0, schedule1)
 f, a, b = schedule_t.get_indices()
 ```
-In this unnatural example, `i0` and `j1` are fused and named `a`, and `i1` and `j0` are fused and named `b`. As before, Accera guarantees that for each value of `a` and `b` the operation `C[a, b] += A[a, b]` is executed before `C[b, a] *= B[b, a]`. As noted above, the fusing operation itself preserves logical equivalence, but if we proceed to transform the fused schedule as follows,
+In this unnatural example, `i0` and `j1` are fused and named `a`. Similarly,`i1` and `j0` are fused and named `b`. As mentioned above, Accera guarantees that, for each value of `a` and `b`, the operation `C[a, b] += A[a, b]` is executed before `C[b, a] *= B[b, a]`. The fusing operation itself preserves the logical equivalence. However, the underlying logic is changed if we transform the fused schedule as follow: 
 ```python
 schedule_t.reorder(a, b, f)
 ```
-the logic actually changes. To see this, note that the resulting schedule is equivalent to the following Python code:
+To understand this change in the logic, note that the resulting schedule is equivalent to the following Python code:
 ```python
 for a in range(16):
     for b in range(16):
         C[a, b] += A[a, b]
         C[b, a] *= B[b, a]
 ```
-In particular, this code sets `C[1,0]` to `C[1,0] * B[1,0] + A[1,0]`, whereas the original fused logic set `C[1,0]` to `(C[1,0] + A[1,0]) * B[1,0] `. We conclude that `schedule_t` is certainly not safe. If the programmer creates an unsafe schedule, they take upon themselves the responsibility of maintaining logical equivalence.
+The above code sets `C[1,0]` to `C[1,0] * B[1,0] + A[1,0]`, whereas the original fused logic set `C[1,0]` to `(C[1,0] + A[1,0]) * B[1,0] `. In this case, we can conclude that `schedule_t` is definitely not safe. If the programmer decides to create an unsafe schedule, Accera can no longer guarantee safety, and they take upon themselves the responsibility of maintaining logical equivalence.
```

Comment:
we can omit "Accera can no longer guarantee safety, and" (implied by the sentence)